src/core/shiftManagement.ts

```diff
@@ -9,14 +9,21 @@ import { prisma } from "../prismaClient";
 
 type Timestamps = "createdAt" | "updatedAt";
 
+type GenericSuccessResponse = {
+  status: "ok";
+};
+
 export function listAllShifts(): Promise<Shift[]> {
   return prisma.shift.findMany();
 }
 
 export function listHealthCareFacilityShifts(
   facilityUuid: HealthCareFacility["uuid"],
 ): Promise<Shift[]> {
-  return prisma.shift.findMany({ where: { facilityUuid } });
+  return prisma.shift.findMany({
+    where: { facilityUuid },
+    include: { shiftAssignments: true },
```

Comment:
🔍 Observation
I notice the PR adds `include: { shiftAssignments: true }` to the `listHealthCareFacilityShifts` function. This change appears to be unnecessary for several reasons:

- Requirements Alignment

The current requirements don't specify a need for shift assignment data in facility shift listing
The new features (rating and blocking) don't require modification to this endpoint

- Performance Impact

Including shiftAssignments will fetch additional data for every shift
This increases both database load and response payload size unnecessarily
For facilities with many shifts and assignments, this could significantly impact response time

Happy to discuss if there are specific use cases I might not be aware of that necessitate this change..